From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Sat, 23 Apr 2022 19:52:19 +0200
Subject: Some changes to octave invocation options, no change in test fail

---
 tests/sh/common.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/sh/common.sh b/tests/sh/common.sh
index a051bd8..5dbda8b 100644
--- a/tests/sh/common.sh
+++ b/tests/sh/common.sh
@@ -16,7 +16,7 @@ trapSCHLD() {
 trap trapSCHLD SIGCHLD
 
 startOctave() {
-    coproc octave --no-init-file -W --no-gui --path $PWD/../../src/m
+    coproc octave --no-init-file --no-window-system --no-history --no-gui --path $PWD/../../src/m
     octpid=$!
     echo "computer" >&p
     read -p FFF
