From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Fri, 18 Oct 2024 20:09:10 +0200
Subject: create instdir, use DESTDIR

---
 src-js/Makefile.am | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index b13f9b1..400c352 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -53,20 +53,16 @@ node_modules = $(addprefix node_modules/, $(node_module_names))
 
 .PRECIOUS: node_modules/%
 
-node_modules/%: node_modules_updated.txt
-
-node_modules:
-	mkdir $@
-
-$(node_modules) node_modules_updated.txt: node_modules Makefile.am
+$(node_modules) node_modules_updated.txt:
 	touch node_modules_updated.txt
 if acond_NPM_PKGS_AVAIL
-	npm link $(node_module_names)
+	true $(foreach f, $(node_module_names), && npm link $(f))
 endif
 
 install-data-local:
 if acond_NPM_PKGS_AVAIL
-	cd $(instdir) $(foreach f, $(node_module_names), && npm link $(f))
+	install -d $(DESTDIR)$(instdir)
+	cd $(DESTDIR)$(instdir) $(foreach f, $(node_module_names), && npm link $(f))
 endif
 	$(INSTALL) -d $(apache_confd)
 	echo "Alias $(server_base)/p2x $(prefix)/share/p2x/js/" > $(apache_confd)/p2xjs.conf
