From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Fri, 18 Oct 2024 19:49:06 +0200
Subject: npm link in DESTDIR

---
 src-js/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index b13f9b1..d47b363 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -66,7 +66,7 @@ endif
 
 install-data-local:
 if acond_NPM_PKGS_AVAIL
-	cd $(instdir) $(foreach f, $(node_module_names), && npm link $(f))
+	cd $(DESTDIR)$(instdir) $(foreach f, $(node_module_names), && npm link $(f))
 endif
 	$(INSTALL) -d $(apache_confd)
 	echo "Alias $(server_base)/p2x $(prefix)/share/p2x/js/" > $(apache_confd)/p2xjs.conf
