From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Fri, 18 Oct 2024 20:40:02 +0200
Subject: Try this

---
 src-js/Makefile.am | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index 47107dc..c66cba4 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -53,15 +53,10 @@ node_modules = $(addprefix node_modules/, $(node_module_names))
 
 .PRECIOUS: node_modules/%
 
-node_modules/%: node_modules_updated.txt
-
-node_modules:
-	mkdir $@
-
-$(node_modules) node_modules_updated.txt: node_modules Makefile.am
+$(node_modules) node_modules_updated.txt:
 	touch node_modules_updated.txt
 if acond_NPM_PKGS_AVAIL
-	npm link $(node_module_names)
+	true $(foreach f, $(node_module_names), && npm link $(f))
 endif
 
 install-data-local:
