From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 30 May 2022 12:52:07 +0200
Subject: Better makefile dependencies to install the npm modules

---
 src-js/Makefile.am | 18 ++++++++++++------
 1 file changed, 12 insertions(+), 6 deletions(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index 76ac9b9..497ae2f 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -12,9 +12,9 @@ unmin_files = $(ncd_files) scanner.js hashmap.js parse-xml.js parse-opts.js p2x.
 
 min_files = $(addsuffix .min.js, $(basename $(unmin_files)))
 
-generated_files = $(min_files) $(json_files)
+generated_files = $(min_files) $(json_files) node_modules_updated.txt
 
-project_files = $(generated_files) $(unmin_files)
+project_files = $(min_files) $(json_files) $(unmin_files)
 
 inst_DATA = $(project_files)
 
@@ -23,10 +23,10 @@ gennc-common.js: token.ncd.js
 %.ncd.js: $(top_srcdir)/src/xml/%.ncd.xml
 	gennc -m JavaScript $^
 
-%.json: $(top_srcdir)/src/xml/%.xml $(node_modules) scanconf-xml2json.min.js
+%.json: $(top_srcdir)/src/xml/%.xml node_modules_updated.txt scanconf-xml2json.min.js $(min_files)
 	./p2xjs scanconf-xml2json -S $< -o $@
 
-%.min.js: %.js  compress.js $(node_modules)
+%.min.js: %.js compress.js node_modules_updated.txt
 	sed -e '/require/ s/.js/.min.js/' $< > $<.tmp
 	nodejs compress.js -o $@ $<.tmp
 	rm $<.tmp
@@ -48,8 +48,14 @@ node_modules = $(addprefix node_modules/, $(node_module_names))
 
 .PRECIOUS: node_modules/%
 
-node_modules/%:
-	npm link $*
+node_modules/%: node_modules_updated.txt
+
+node_modules:
+	mkdir $@
+
+$(node_modules) node_modules_updated.txt: node_modules Makefile.am
+	touch node_modules_updated.txt
+	npm link $(node_module_names)
 
 $(instdir):
 	$(INSTALL) -d $@
