From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Fri, 18 Oct 2024 20:40:02 +0200
Subject: Try this

---
 src-js/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index 47107dc..15fef2b 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -61,7 +61,7 @@ node_modules:
 $(node_modules) node_modules_updated.txt: node_modules Makefile.am
 	touch node_modules_updated.txt
 if acond_NPM_PKGS_AVAIL
-	npm link $(node_module_names)
+	true $(foreach f, $(node_module_names), && npm link $(f))
 endif
 
 install-data-local:
