From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Fri, 18 Oct 2024 20:45:02 +0200
Subject: configure

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index cc22026..e044332 100644
--- a/configure.ac
+++ b/configure.ac
@@ -33,7 +33,7 @@ if [test "$js_enabled" = "yes"]; then
 AC_CHECK_PROGS([NPM], [npm])
 
 if [test "x$NPM" != "x"]; then
-   npm_packages="most-xml tmp node-minify"
+   npm_packages="node-minify most-xml tmp"
    npm_packages_avail=yes
    for pkg in $npm_packages; do
        if ! $NPM list -g $pkg > /dev/null; then
