From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Sun, 20 Oct 2024 17:40:57 +0200
Subject: Print bad test logs

---
 tests/sh/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/sh/Makefile.am b/tests/sh/Makefile.am
index 363005a..31e771f 100644
--- a/tests/sh/Makefile.am
+++ b/tests/sh/Makefile.am
@@ -22,6 +22,7 @@ bad_test_results = $(addprefix result_suite_bad_,$(bad))
 
 result_suite_good_%: %.sh
 	./$< > log_$*.txt 2>&1
+	if [[ $$? != 0 ]]; then cat log_$*.txt; fi
 
 result_suite_bad_%: %.sh
 	! ./$< > log_$*.txt 2>&1
