From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Thu, 2 Jun 2022 23:21:34 +0200
Subject: configure switch for js version

---
 configure.ac | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/configure.ac b/configure.ac
index 5c10189..00f1ec0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -33,6 +33,15 @@ AM_CONDITIONAL(acond_GENGETOPT_AVAIL, [test "x$GENGETOPT" != "x"])
 AC_CHECK_PROGS([NPM], [npm])
 AM_CONDITIONAL(acond_NPM_AVAIL, [test "x$NPM" != "x"])
 
+AC_ARG_ENABLE([js], [  --enable-js    Enable install of P2X JS implementation.
+  --disable-js   No VCS version file.], [
+      js_enabled=$enableval
+], [
+      js_enabled=yes
+])
+AM_CONDITIONAL(acond_JS_ENABLED, [test "$js_enabled" = "yes"])
+
+if [test "$js_enabled" = "yes"]; then
 if [test "x$NPM" != "x"]; then
    npm_packages="most-xml tmp node-minify"
    npm_packages_avail=yes
@@ -44,6 +53,7 @@ if [test "x$NPM" != "x"]; then
        fi
    done
 fi
+fi
 AM_CONDITIONAL(acond_NPM_PKGS_AVAIL, [test "$npm_packages_avail" != ""])
 
 AC_CHECK_PROGS([TRANG], [trang])
@@ -243,9 +253,11 @@ echo " shunit2:        [$SHUNIT2]"
 echo " valgrind:       [$VALGRIND]"
 fi
 echo ""
+if [test "$js_enabled" = "yes"]; then
 echo " npm:            [$NPM]"
 echo " npm packages:   [$npm_packages_avail]"
 if test "$build_tests" = "yes"; then
 echo " mocha:          [$MOCHA]"
 fi
 echo ""
+fi
