From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Sun, 20 Oct 2024 18:13:14 +0200
Subject: JS tests only work when most-xml is installed

---
 tests/sh/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/sh/Makefile.am b/tests/sh/Makefile.am
index 204b19e..112df29 100644
--- a/tests/sh/Makefile.am
+++ b/tests/sh/Makefile.am
@@ -11,7 +11,7 @@ good += val_tests test_reproduce_strict test_reproduce test_reparse	\
  output_mode_matlab cdecls
 endif
 
-if acond_NPM_AVAIL
+if acond_NPM_PKGS_AVAIL
 good += p2xjs_mocha_tests test_cmp_js reproduce_ign
 endif
 
