From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Wed, 1 Jun 2022 23:24:19 +0200
Subject: Install to correct location

---
 src-js/Makefile.am | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index 62e9cc6..2ebb0e7 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -1,7 +1,7 @@
 
 json_files = scanner-c.json
 
-instbasedir = $(DESTDIR)$(pkgdatadir)
+instbasedir = $(pkgdatadir)
 instdir = $(instbasedir)/js
 
 dist_bin_SCRIPTS = p2xjs
@@ -64,10 +64,7 @@ if acond_NPM_PKGS_AVAIL
 	npm link $(node_module_names)
 endif
 
-$(instdir):
-	$(INSTALL) -d $@
-
-install-data-local: $(instdir)
+install-data-local:
 if acond_NPM_PKGS_AVAIL
 	cd $(instdir) $(foreach f, $(node_module_names), && npm link $(f))
 endif
