From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 16:49:24 +0200
Subject: Fix: whitespace not ignored

---
 examples/xsl/cdecls.xsl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/xsl/cdecls.xsl b/examples/xsl/cdecls.xsl
index e2547b4..506f6cc 100644
--- a/examples/xsl/cdecls.xsl
+++ b/examples/xsl/cdecls.xsl
@@ -5,7 +5,7 @@ Copyright © 2014,2016 Johannes Willkomm
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
     xmlns:cx='http://johannes-willkomm.de/xml/code-xml/'
     xmlns:c='http://johannes-willkomm.de/xml/code-xml/attributes/'
-    xmlns:ci='http://johannes-willkomm.de/xml/code-xml/ignore'
+    xmlns:ci='http://johannes-willkomm.de/xml/code-xml/ignore/'
     version="1.0">
 
   <xsl:output method="text"/>
