From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Sat, 19 Oct 2024 15:44:43 +0200
Subject: Try twice

---
 src-js/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src-js/Makefile.am b/src-js/Makefile.am
index ebd61db..2bb10bc 100644
--- a/src-js/Makefile.am
+++ b/src-js/Makefile.am
@@ -63,6 +63,7 @@ endif
 install-data-local:
 if acond_NPM_PKGS_AVAIL
 	install -d $(DESTDIR)$(instdir)
+	-cd $(DESTDIR)$(instdir) $(foreach f, $(node_module_names), && npm link $(f))
 	cd $(DESTDIR)$(instdir) $(foreach f, $(node_module_names), && npm link $(f))
 endif
 	$(INSTALL) -d $(apache_confd)
