From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 16:53:01 +0200
Subject: Add cdecls to list of tests

---
 tests/sh/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/sh/Makefile.am b/tests/sh/Makefile.am
index 777d1ae..2fa02d7 100644
--- a/tests/sh/Makefile.am
+++ b/tests/sh/Makefile.am
@@ -4,7 +4,7 @@ generated_files = all_tests.sh test_cmp_js.sh test_cmp_jsy.sh test_reparse.sh te
 good = val_tests test_reproduce_strict test_reproduce test_reparse	\
 parse_tests merge_tests p2x_tests test_output_stability indent_tests	\
 valgrind_tests output_mode_json_merged output_mode_json			\
-output_mode_matlab_merged output_mode_matlab
+output_mode_matlab_merged output_mode_matlab cdecls
 
 # js tests
 # reproduce_ign test_cmp_js p2xjs_mocha_tests
