From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 16:53:48 +0200
Subject: Ignore autogen test file

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 1269c4f..e745524 100644
--- a/.gitignore
+++ b/.gitignore
@@ -20,6 +20,7 @@ tests/sh/test_*.sh
 src-js/node_modules
 src-js/scanner-c.json
 tests/sh/forall_tests.sh
+tests/sh/all_tests.sh
 tests/sh/log_*.txt
 tests/sh/err_*.txt
 src-js/p2xa.js
