From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 17:07:48 +0200
Subject: cdecls test

---
 tests/sh/cdecls.sh | 20 ++++++++++++++++----
 1 file changed, 16 insertions(+), 4 deletions(-)

diff --git a/tests/sh/cdecls.sh b/tests/sh/cdecls.sh
index 19de03d..b5e7462 100755
--- a/tests/sh/cdecls.sh
+++ b/tests/sh/cdecls.sh
@@ -1,16 +1,28 @@
 #! /bin/zsh
 
 export SHUNIT_PARENT=$0
-. ./setup_zsh.sh
+
+. ./setup.sh
+
+setUp() {
+    :
+}
+
+tearDown() {
+    :
+}
 
 parseCCode() {
     inf=$1
-#    p2x -gmp ../../examples/configs/cdecls $inf | xsltproc ../../src/xsl/parens.xsl -
-    p2x -gmXp ../../examples/configs/cdecls $inf | xsltproc ../../examples/xsl/cdecls.xsl -
+    p2x -gmXp $exdir/configs/cdecls $inf | tee cdecls.xml | xsltproc $exdir/xsl/cdecls.xsl -
+    ps=($pipestatus)
+    assertEquals "${ps[1]}" "0"
+    assertEquals "${ps[2]}" "0"
+    assertEquals "${ps[3]}" "0"
 }
 
 test_cdecls1() {
-    parseCCode ../../examples/in/cdecls.exp
+    parseCCode $exdir/in/cdecls.exp
 }
 
 . ./myshunit2
