From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 17:22:25 +0200
Subject: Test

---
 examples/Makefile.am | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/examples/Makefile.am b/examples/Makefile.am
index a414bc3..9613af6 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -8,12 +8,13 @@ instbasedir=$(docdir)
 instdir= $(instbasedir)/examples
 
 install:
+	echo "prefix=$(prefix)"
 	mkdir -p $(instdir)/configs
 	mkdir -p $(instdir)/in
 	mkdir -p $(instdir)/xsl
-	/usr/bin/install -c -m 644 configs/cdecls $(instdir)/configs
-	/usr/bin/install -c -m 644 in/cdecls.exp  $(instdir)/in
-	/usr/bin/install -c -m 644 xsl/cdecls.xsl $(instdir)/xsl
+	install -c -m 644 configs/cdecls $(instdir)/configs
+	install -c -m 644 in/cdecls.exp  $(instdir)/in
+	install -c -m 644 xsl/cdecls.xsl $(instdir)/xsl
 
 examples = $(wildcard in/*.exp)
 
