From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Mon, 23 May 2022 22:06:36 +0200
Subject: Another tweak

---
 tests/run-unit-test | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/run-unit-test b/tests/run-unit-test
index 8b92eb7..d87cef3 100644
--- a/tests/run-unit-test
+++ b/tests/run-unit-test
@@ -12,8 +12,9 @@ cd $AUTOPKGTEST_TMP
 cp -a $P2X_HOME/share/doc/$debname/tests/* $AUTOPKGTEST_TMP
 gunzip -r *
 for testfile in *.sh; do
-    if head -n 6 $testfile | grep setup.sh &> /dev/null; then
+    if head -n 6 $testfile | grep setup.sh > /dev/null 2>&1; then
         echo "BEGIN TEST $testfile"
+        chmod a+x $testfile
         ./$testfile
     fi
 done
